src/date.spec.js

```diff
@@ -565,39 +578,44 @@ describe('uiDateFormat', function() {
 
     it('use ISO if not config value', function() {
       inject(['$compile', '$rootScope', function($compile, $rootScope) {
-        element = $compile('<input ui-date-format ng-model="x"/>')($rootScope);
+        element = $compile('<input ui-date ui-date-format ng-model="x"/>')($rootScope);
         scope = $rootScope;
       }]);
 
       var aDate = new Date(2012, 9, 11);
       var aISODateString = aDate.toISOString();
       scope.x = aISODateString;
       scope.$digest();
-      expect(element.controller('ngModel').$viewValue).toEqual(aDate);
-    });
-
-    it('use format value if config given', function() {
-      var format = 'yy DD, d MM';
-      module(function($provide) {
-        $provide.constant('uiDateFormatConfig', format);
-      });
-
-      inject(['$compile', '$rootScope', function($compile, $rootScope) {
-        element = $compile('<input ui-date-format ng-model="x"/>')($rootScope);
-        scope = $rootScope;
-      }]);
-
-      var aDateString = '2012 Friday, 12 October';
-      var expectedDate = new Date('2012-10-12');
-
-      scope.x = aDateString;
-      scope.$digest();
 
-      var pickerDate = element.controller('ngModel').$viewValue;
-      expect(pickerDate.getDate()).toEqual(expectedDate.getUTCDate());
-      expect(pickerDate.getUTCMonth()).toEqual(expectedDate.getUTCMonth());
-      expect(pickerDate.getUTCFullYear()).toEqual(expectedDate.getUTCFullYear());
+      var dateFormat = element.datepicker('option', 'dateFormat');
+      var dateObj = $.datepicker.formatDate(dateFormat, aDate);
+      expect(element.controller('ngModel').$viewValue).toEqual(dateObj);
     });
+
+    //Logic change - no more uiDateFormatConfig constant used, we could set default $.datepicker globally or change
```

Comment:
Some people also have used this constant in the past, not sure if we can safely deprecate.  Do we really need to stop using this global config?